specifyweb/frontend/js_src/lib/components/SchemaConfig/Tables.tsx

```diff
@@ -85,6 +90,7 @@ export function tablesFilter(
   selectedTables: RA<keyof Tables> | undefined = undefined
 ): boolean {
   if (selectedTables?.includes(name) === true) return true;
+  if (tablesToIncludeInQueryList.has(name)) return true;
```

Comment:
"tablesToIncludeInQueryList" but this function is used in more than query list...
it's used everywhere were tables are shown in specify

instead, this function has a `selectedTables` list that you can use to allow list certain tables, regardless of all other filters

Reply:
then, whether to force-include hierarchy tables like Collection/Discipline/Divison depends on the use case - is the use case for user to be able to edit them? if so, you should probably only include them if the user has edit permission to them.

or you could simply say that if user is a super admin then show all hidden tables even if not explicitly said so in the user preferences

Reply:
or do you want these tables to be included in all lists across specify (in workbench mapper too and everywhere else?). if so, we need to revisit the definition of system tables. a system table seemed to be before a table that user ordinary has no need to touch - if that is not the case for these tables, then should we mark them as not system?

or is there a reason to keep them as system on the back-end? in that case you can mark them as non-system on the front-end only in this file:

https://github.com/specify/specify7/blob/issue-5138/specifyweb/frontend/js_src/lib/components/DataModel/schemaOverrides.ts

Reply:
yes those 3 can be included in all places. Changing the isSystem in the schemaOverrides creates a lot of issues in other parts of sp and it doesn't worth the trouble for just 3 tables to add in a list, hence our choice to go with this short list variable.

Reply:
the other thing is that we do not want all the tables but only those 4 which would not be the case if we use the isAdmin check right?

Reply:
maybe adding the tables for admin managers? (pushed a test below)